### DISCUSSION TITLE
what could be causing this pnpm build command error?

### DISCUSSION THREAD

**Discussion by matthewhausman** [wevm/wagmi#1217] [General]
Generating TypeScript declarations for packages/vue/src/hooks/accounts/useAccount.ts failed:

The inferred type of 'useAccount' cannot be named without a reference to '../../../../core/node_modules/eventemitter3'. This is likely not portable. A type annotation is necessary.
🎁 error
🎁 error 31 export const useAccount = ({

**Comment by dimovski-dev** [comment]
+1
